### ISSUE TITLE
[Bug Report][3.5.16] VTimePicker am/pm button cannot be clicked

### ISSUE THREAD

**Issue by kingyue737** [vuetifyjs/vuetify#19637]
### Environment
**Vuetify Version:** 3.5.16
**Vue Version:** 3.4.23
**Browsers:** Edge 123.0.0.0
**OS:** Windows 10

### Steps to reproduce
Click am/pm button

### Expected Behavior
Can be clicked

### Actual Behavior
Cannot be clicked

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpNi8EKgCAQRH9FPFd7DxH6juogsgdJZdHVS/TviRJ0GeYxb/Zb5mRhI1pqQblKxRjIG0Z9RCFUnQ1Rrx3YBZzJ2QuTgGHApyj4XRtmmxyxyMiFtIKBbZLPFIv3Pc4XwmIsBw==)

### Other comments
am/pm button group has class 'v-time-picker-controls__ampm--readonly' which disables pointer events

**Comment by kingyue737** [comment]
Required to `ampm-in-title` to make it not read-only. Is this the expected behavior?

https://github.com/vuetifyjs/vuetify/blob/afee26200e937659d69db68c29a5d8d40b9ee7cd/packages/vuetify/src/labs/VTimePicker/VTimePicker.tsx#L314

**Comment by kiri10ten** [comment]
Hello @kingyue737  to temporarly fetch the AM and PM within the v-model  ,what am I supposed to do  ! I would be grateful If you could answer it ,Thank you !

**Comment by rktaxali** [comment]
Is there an easy fix to make AM/PM clickable?

**Comment by kingyue737** [comment]
@rktaxali just set `ampmInTitle`

```vue
<v-time-picker anpm-in-title />
```
or config globally
```ts
createVuetify({
  defaults: {
    VTimePicker: {
      ampmInTitle: true,
    },
  }
})
```

**Comment by blalan05** [comment]
Yes, the bug is that the AM/PM selector shouldn't be shown, which is why they are disabled.
